_posts/2016-05-25-hybox-at-or.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Find out What’s Inside Hydra-in-a-Box at Open Repositories 2016 - PCDM, Design, Emerging Architecture, Repository Tooling
+---
+
+It’s only three weeks away! If you will attend the 11th Annual International Conference on Open Repositories (#OR2016), here are the sessions that will be of interest if you want to learn more about the Hydra-in-a-Box project.
```

Comment:
I believe the conference planners want folks to use `#OR2016dub` to avoid commingling social media traffic with other "OR"s.